### PR TITLE
Support specifying a dtk binary

### PR DESCRIPTION
**Pull request by ribbanya**
From doldecomp/melee#1335.

### CODE REVIEW

**Comment by encounter** [comment]
For consistency with the other arguments, let's rename this to just `--dtk`

**Comment by ribbanya** [comment]
How's this?
```
❯ ./configure.py --help
usage: configure.py [-h] [--version VERSION] [--build-dir BUILD_DIR] [--compilers DIR] [--map] [--debug]
                    [--wrapper BINARY] [--dtk BINARY | SOURCE_DIR] [--sjiswrap SJISWRAP] [--verbose]
                    [mode]

positional arguments:
  mode                  configure or progress (default: configure)

options:
  -h, --help            show this help message and exit
  --version VERSION     version to build (GAMEID)
  --build-dir BUILD_DIR
                        base build directory (default: build)
  --compilers DIR       path to compilers (optional)
  --map                 generate map file(s)
  --debug               build with debug info (non-matching)
  --wrapper BINARY      path to wibo or wine (optional)
  --dtk BINARY | SOURCE_DIR
                        path to decomp-toolkit source or binary (optional)
  --sjiswrap SJISWRAP   path to sjiswrap.exe (optional)
  --verbose             print verbose output
```

**Comment by ribbanya** [comment]
Missed a metavar.

```
❯ ./configure.py --help
usage: configure.py [-h] [--version VERSION] [--build-dir BUILD_DIR] [--compilers DIR] [--map] [--debug]
                    [--wrapper BINARY] [--dtk BINARY | SOURCE_DIR] [--sjiswrap EXE] [--verbose]
                    [mode]

positional arguments:
  mode                  configure or progress (default: configure)

options:
  -h, --help            show this help message and exit
  --version VERSION     version to build (GAMEID)
  --build-dir BUILD_DIR
                        base build directory (default: build)
  --compilers DIR       path to compilers (optional)
  --map                 generate map file(s)
  --debug               build with debug info (non-matching)
  --wrapper BINARY      path to wibo or wine (optional)
  --dtk BINARY | SOURCE_DIR
                        path to decomp-toolkit source or binary (optional)
  --sjiswrap EXE        path to sjiswrap.exe (optional)
  --verbose             print verbose output
```

File: configure.py [hunk 1]
```diff
@@ -109,7 +106,7 @@
 
 # Apply arguments
 config.build_dir = args.build_dir
-config.build_dtk_path = args.build_dtk
+config.build_dtk_path = args.dtk
```
Comment (encounter) [inline configure.py:109]: Let's rename this config field too

Comment (ribbanya) [reply configure.py]: Like f146c738771fdc679af2bea62ac0b1969f7d4a11?

Comment (ribbanya) [reply configure.py]: Oh, it should be `_path`, I see.

**Comment by ribbanya** [comment]
I think I'm done committing for real now.

```
❯ ./configure.py --help             
usage: configure.py [-h] [--version {GAMEID}] [--build-dir DIR] [--binutils BINARY] [--compilers DIR] [--map]
                    [--debug] [--wrapper BINARY] [--dtk BINARY | DIR] [--sjiswrap EXE] [--verbose]
                    [{configure,progress}]

positional arguments:
  {configure,progress}  script mode (default: configure)

options:
  -h, --help            show this help message and exit
  --version {GAMEID}    version to build
  --build-dir DIR       base build directory (default: build)
  --binutils BINARY     path to binutils (optional)
  --compilers DIR       path to compilers (optional)
  --map                 generate map file(s)
  --debug               build with debug info (non-matching)
  --wrapper BINARY      path to wibo or wine (optional)
  --dtk BINARY | DIR    path to decomp-toolkit binary or source (optional)
  --sjiswrap EXE        path to sjiswrap.exe (optional)
  --verbose             print verbose output
```

**Comment by encounter** [comment]
Looks great! I just caused you another merge conflict, sorry 🙃

**Comment by ribbanya** [comment]
Resolved.

**Pull request merged**